### PR TITLE
added instance to subject line in notifications

### PR DESCRIPTION
**Pull request by ives1227**
**Adds the instance to the subject line for distinguishing emails*
* * *

**Ticket Link**: None

No tests - just a small line of code

### CODE REVIEW

File: app/notifier/dais_notifier.py [hunk 1]
```diff
@@ -39,7 +39,8 @@ def __create_message(self, email_subject, email_body, email_destination):
         message = EmailMessage()
         message.set_content(email_body)
 
-        message['Subject'] = email_subject
+        instance = os.getenv("ENV", "instance unknown")
```
Comment (awoods) [inline app/notifier/dais_notifier.py:42]: Does case matter? We seem to be using lowercase: https://github.huit.harvard.edu/LTS/DAIS-IF/blob/4d4288d5a7cef6e3dad35c3d77df721060f692ca/environments/qa/group_vars/all/main.yml#L8

File: app/notifier/dais_notifier.py [hunk 2]
```diff
@@ -39,7 +39,8 @@ def __create_message(self, email_subject, email_body, email_destination):
...
+        message['Subject'] = "(" + instance + ")" + email_subject
```
Comment (awoods) [inline app/notifier/dais_notifier.py:43]: For readability, please add as space after closing ')'.

**Review by awoods — CHANGES_REQUESTED** [review]

File: app/notifier/dais_notifier.py (hunk 2 above)
Comment (ives1227) [reply app/notifier/dais_notifier.py]: Done

File: app/notifier/dais_notifier.py (hunk 1 above)
Comment (ives1227) [reply app/notifier/dais_notifier.py:42]: @awoods it is capitalized in the .env: https://github.huit.harvard.edu/LTS/DAIS-IF/blob/4d4288d5a7cef6e3dad35c3d77df721060f692ca/roles/deploy/templates/notifier/.env#L2. The lowercase is just for the template value.

**Review by awoods — APPROVED** [review]

**Pull request merged**